source/assets/js/playground/utils.ts

```diff
@@ -35,7 +35,10 @@ export function serializeState(state: PlaygroundState): string {
  * - `inputFormat`: 0=indented 1=scss
  * - `outputFormat`: 0=compressed 1=expanded
  */
-function serializeStateContents(state: PlaygroundState): string {
+export function serializeStateContents(
+  state: Partial<PlaygroundState> &
+    Pick<PlaygroundState, 'inputFormat' | 'outputFormat' | 'inputValue'>
```

Comment:
I think this could be simplified?

```suggestion
  state: Pick<PlaygroundState, 'inputFormat' | 'outputFormat' | 'inputValue'>
```

Reply:
Hmm, that was so the function could be called with the whole state or a subset, but I can't track down where the whole state was being passed, so I took your suggestion.

Reply:
Oh, makes sense. I think the entire state is passed in earlier in this same file, but I also think TS allows _extra_ properties on objects as long as the specified ones match.